src/ui_parts/main_scene.tscn

```diff
@@ -51,3 +48,5 @@ layout_mode = 2
 [node name="Display" parent="HSplitContainer" instance=ExtResource("3_qbqbs")]
 unique_name_in_owner = true
 layout_mode = 2
+size_flags_horizontal = 3
+size_flags_stretch_ratio = 1.5
```

Comment:
i removed split offset and made it use this stretch ratio instead, i hope its fine

Reply:
I'd like for the viewport to be exactly square by default, does this affect that?

Reply:
oh well, probably 💀